docpages/example_programs/the_basics/editing-plural.md

```diff
@@ -1,8 +1,8 @@
-\page editing-channel-and-message Editing Channel and Message
+\page editing-plural Editing Plural
```

Comment:
What? This isn't even what we discussed, this is far from it. Have you rushed this again?

Change it to `editing-channels-and-messages` and `Editing Channels and Messages`.

Plural means multiple, more than one. If you make a word plural, you make it mean more than one. So like making the word "Apple" plural means you change it to "Apples".